### ISSUE TITLE
Extra question re knowledge security and safety researchers

### ISSUE THREAD

**Issue by djhcapel** [DH-IT-Portal-Development/ethics#546]
Could we add an extra question under the question about ethical reflection?

Zijn er kwesties rondom kennisveiligheid en/of de veiligheid van de onderzoeker(s)? ja/nee, zo ja, licht toe.

**Comment by djhcapel** [comment]
Under the question about ethical reflection?

**Comment by tymees** [comment]
Right.... Sorry, long day

**Comment by djhcapel** [comment]
Discussed in GC meeting: make this two separate questions:

- Zijn er kwesties rondom de veiligheid van de onderzoeker(s) en deelnemers? ja/nee, zo ja, licht toe.

- Zijn er kwesties rondom kennisveiligheid? ja/nee, zo ja, licht toe.

**Comment by djhcapel** [comment]
Helptekst kennisveiligheid:
Kennisveiligheid gaat over het tijdig signaleren en mitigeren van veiligheidsrisico’s bij wetenschappelijk onderzoek. Voor meer informatie, zie https://intranet.uu.nl/kennisbank/kennisveiligheid.

Knowledge security is about the detection and mitigation of security risks in academic research in a timely manner. For more information, see https://intranet.uu.nl/en/knowledgebase/knowledge-security.

**Comment by djhcapel** [comment]
I just noticed that at the end of the form, there are already some related questions about risks, misleading etc. I will discuss this issue with the chairs and let you know.

**Comment by djhcapel** [comment]
Yes, the two questions should be added as first questions before the question: "Is er binnen dit onderzoekstraject sprake van misleiding van de deelnemer?"

**Comment by djhcapel** [comment]
See email of 26 February with attached the revised text and some additional questions.

**Comment by EdoStorm96** [comment]
So, as of PR #625, all these questions are asked on study end, which means researchers are asked these new questions for every study/trajectory. Today we decided this is mostly fine, but that the question about knowledge security should be asked about the entire proposal/application. This could be on the first page, the submit page, or a new page. We still are not 100% sure about this.

**Comment by djhcapel** [comment]
I have discussed the location of the following questions with the chairs of the FEtC-H and we agree that these two questions should appear at the end, as questions about the entire research, not just one trajectory:

•	Zijn er kwesties rondom kennisveiligheid? ja/nee/twijfel, (bij ja/twijfel: licht toe).
•	Zijn er kwesties rondom de veiligheid van / risico’s voor de onderzoeker(s) ja/nee/twijfel, (bij ja/twijfel: licht toe).

Should we discuss where exactly these should appear?

**Comment by EdoStorm96** [comment]
So I was thinking:
- we rename the page now named "Overzicht" aka `study_end` to "Overzicht traject".
- then we add a page after the study named "afronding trajecten" or "complete trajectories". Here we then ask the questions about knowledge security and researcher safety.

What do you think of that @djhcapel?

**Comment by djhcapel** [comment]
Yes, the page that is now entitled "Overzicht en eigen beoordeling van het gehele onderzoek" should be entitled "Overzicht en eigen beoordeling Traject [x]'.

I think the extra page should be added at the end of 'Trajectories', in the menu 'Overzicht gehele onderzoek', so before 'Formulieren'. Is that possible?

And can this page include brief info on the Trajectories as well?

**Comment by EdoStorm96** [comment]
What would you like to see included in the overview?
I guess we could do a small table with:

- Trajectory 1 | [title] | [research type(s)]
- Trajectory 2 | [title] | [research type(s)]

Maybe add age groups?

Also, do you have an idea for a small text on this page? Right now I have:
"Onderstaande vragen hebben betrekking op het gehele onderzoek."

**Comment by djhcapel** [comment]
I think the suggested overview will do. This should not be too long, because otherwise the questions will not get the attention they deserve. 

The suggested text is also fine.
